src/outfit.ts

```diff
@@ -340,6 +342,108 @@ function cheeses(embezzlerUp: boolean) {
       )
     : [];
 }
+
+function mafiaThumbRing(equipMode: BonusEquipMode) {
+  if (!have($item`mafia thumb ring`) || ["free", "dmt"].some((mode) => mode === equipMode)) {
+    return new Map<Item, number>([]);
+  }
+
+  return new Map<Item, number>([
+    [$item`mafia thumb ring`, (1 / 0.96 - 1) * get("valueOfAdventure")],
+  ]);
+}
+
+function luckyGoldRing(equipMode: BonusEquipMode) {
+  // Ignore for DMT, assuming mafia might get confused about the volcoino drop by the weird combats
+  if (!have($item`lucky gold ring`) || ["dmt"].some((mode) => mode === equipMode)) {
+    return new Map<Item, number>([]);
+  }
+
+  // Volcoino has a low drop rate which isn't accounted for here
+  // Overestimating until it drops is probably fine, don't @ me
+  const dropValues = [
+    100, // 80 - 120 meat
+    ...[
+      itemAmount($item`hobo nickel`) > 0 ? 100 : 0, // This should be closeted
+      itemAmount($item`sand dollar`) > 0 ? getSaleValue($item`sand dollar`) : 0, // This should be closeted
+      itemAmount($item`Freddy Kruegerand`) > 0
+        ? Math.max(
+            getSaleValue($item`bottle of Bloodweiser`) / 200,
+            getSaleValue($item`electric Kool-Aid`) / 200
+          )
+        : 0,
+      get("sleazeAirportAlways") || get("_sleazeAirportToday")
+        ? getSaleValue($item`one-day ticket to Spring Break Beach`) / 100
+        : 0,
+      get("spookyAirportAlways") || get("_spookyAirportToday")
+        ? Math.max(
+            getSaleValue($item`one-day ticket to Conspiracy Island`) / 100,
+            getSaleValue($item`karma shawarma`) / 7
+          )
+        : 0,
+      get("stenchAirportAlways") || get("_stenchAirportToday")
+        ? getSaleValue($item`one-day ticket to Dinseylandfill`) / 20
+        : 0,
+      (get("hotAirportAlways") || get("_hotAirportToday")) && !get("_luckyGoldRingVolcoino")
+        ? getSaleValue($item`one-day ticket to That 70s Volcano`) / 3
+        : 0,
+      get("coldAirportAlways") || get("_coldAirportToday")
+        ? getSaleValue($item`one-day ticket to The Glaciest`) / 50
+        : 0,
+      get("frAlways") || get("_frToday") ? getSaleValue($item`FantasyRealm guest pass`) / 350 : 0,
+    ].filter((value) => value > 0),
+  ];
+
+  // Items drop every ~10 turns
+  return new Map<Item, number>([
+    [
+      $item`lucky gold ring`,
+      dropValues.reduce((previous, current) => previous + current) / dropValues.length / 10,
```

Comment:
```suggestion
      sumNumbers(dropValues) / dropValues.length / 10,
```

importing `sumNumbers` from Libram